plugins/CoreHome/stylesheets/layout.less

```
@@ -190,7 +190,7 @@ nav {
       padding-bottom: 20px;
 
       .menu-icon {
-        padding-right: 13px;
+        padding-right: 9.5px;
```

I guess this one, will be there now in every branch, that's not freshly branched from `4.x-dev`
```suggestion
        padding-right: 13px;
```